riot/runtime/core/proc_effect.ml

```diff
@@ -2,7 +2,7 @@ open Util
 
 type _ Effect.t +=
   | Receive : {
-      ref : 'a Ref.t option;
+      ref : 'a Symbol.t option;
```

Comment:
@leostera I assume we want to change this `ref` to be `symbol` as well? Just want to confirm.